lib/ridgepole/diff.rb

```diff
@@ -30,7 +30,8 @@ def diff(from, to, options = {})
           @logger.verbose_info("#   #{table_name}")
 
           unless @options[:drop_table_only]
-            unless (attrs_delta = diff_inspect(from_attrs, to_attrs)).empty?
+            unless hash_deep_equal?(from_attrs, to_attrs)
+              attrs_delta = diff_inspect(from_attrs, to_attrs)
               @logger.verbose_info(attrs_delta)
```

Comment:
Or perhaps this `verbose_info` call could me moved after `scan_change`, eliminating the need of `hash_deep_equal?` ?